packages/webpack-config/jest-puppeteer.config.js

```diff
@@ -0,0 +1,37 @@
+const launch = process.env.CI
+  ? {
+      args: ['--ignore-certificate-errors', '--no-sandbox', '--disable-setuid-sandbox'],
+      ignoreHTTPSErrors: true,
+      headless: true,
+    }
+  : {
+      args: ['--ignore-certificate-errors'],
+      ignoreHTTPSErrors: true,
+      headless: true,
+    };
+
+const config = {
+  start: {
+    url: 'https://localhost:19006',
+    launch,
+    server: {
+      command: `../expo-cli/bin/expo.js start tests/basic/ --web-only --non-interactive --https`,
+      port: 19006,
```

Comment:
Since jest-dev-server will wait until a server launches on this port, it'd be preferable to explicitly specify the port to Expo CLI so that we are guaranteed to get the same port every time.